spec.md

```diff
@@ -245,6 +243,32 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD chose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are valid:
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| error | string | A single word uniquely identifying the error condition. |
+| description | string | A meaningful error message explaining why the request failed. | 
+
+### Error Codes
+
+There are failure scenarios described throughout this specification for which
+the `error` field MUST contain a specific string. Service Broker authors MUST
```

Comment:
This MUST could be a breaking change.

Reply:
The MUST was already in use throughout the spec, we just consolidated the meaning of the error codes, and they continue to be called out in certain situations as required, for `AsyncRequired` for example, the broker must report that back to the platform if it requires the `accepts_incomplete=true` parameter. 

What we have done is centralized the definition of the error codes and removed some confusing language around calling out what exactly needs to be returned further down in the spec.